src/api/app/lib/backend/test/tasks.rb

```diff
@@ -3,39 +3,15 @@ class Test
     # Module that holds methods for running the different parts of the backend server
     module Tasks
       def run_scheduler(arch)
-        Rails.logger.debug { "RUN_SCHEDULER #{arch}" }
-        perlopts = "-I#{Rails.root}/../backend -I#{Rails.root}/../backend/build"
-        IO.popen("cd #{backend_config}; exec perl #{perlopts} ./bs_sched --testmode #{arch}") do |io|
-          # just for waiting until scheduler finishes
-          io.each { |line| Rails.logger.debug { "scheduler(#{arch}): #{line.strip.chomp}" } if line.present? }
-        end
-      end
-
-      def run_dispatcher
```

Comment:
This method is now gone, is this what we want? I don't think so, could you confirm?